### PR TITLE
Adding affiliated faculty at CU Boulder

### PR DESCRIPTION
**Pull request by lindelph**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -9002,6 +9003,7 @@ Manuel Carro,IMDEA Software Institute,http://software.imdea.org/people/manuel.ca
 Manuel Chakravarty,UNSW,http://www.cse.unsw.edu.au/~chak,w6XYY_EAAAAJ
 Manuel E. Acacio,University of Murcia,http://www.ditec.um.es/~meacacio,iHD6MsUAAAAJ
 Manuel E. Bermudez,University of Florida,http://www.cise.ufl.edu/~manuel,xjUqBT0AAAAJ
+Manuel E. Lladser,University of Colorado Boulder,http://amath.colorado.edu/faculty/lladser/,Uv6cAEMAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:9006]: Not listed here: https://www.colorado.edu/cs/faculty and does not mention an appointment with CS. Please clarify how he meets the inclusion criteria.

File: csrankings.csv [hunk 2]
```diff
@@ -10767,6 +10769,8 @@ Niranjan Balasubramanian,Stony Brook University,http://www3.cs.stonybrook.edu/~n
 Nirmala Shenoy,Rochester Institute of Technology,https://www.rit.edu/gccis/nirmala-shenoy,NOSCHOLARPAGE
 Nirupam Roy,University of Maryland - College Park,https://www.cs.umd.edu/~nirupam,kKyEuTcAAAAJ
 Nirupama Bulusu,Portland State University,http://web.cecs.pdx.edu/~nbulusu,xpyAqbUAAAAJ
+Nisar Ahmed,University of Colorado Boulder,https://cohrint.info/,HA3R65IAAAAJ
+Nisar R. Ahmed,University of Colorado Boulder,https://cohrint.info/,HA3R65IAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:10773]: Same issue as above. https://www.colorado.edu/cs/faculty, no affiliation with CS mentioned on homepage.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (lindelph) [reply csrankings.csv:9006]: That page is unfortunately incomplete. The affiliation is stated here: https://experts.colorado.edu/display/fisid_134170

File: csrankings.csv (hunk 2 above)
Comment (lindelph) [reply csrankings.csv:10773]: And https://experts.colorado.edu/display/fisid_153237 for Nisar.

Comment (lindelph) [reply csrankings.csv:10773]: And to make the case even stronger, Nisar is currently the sole advisor of [this](https://bisraelsen.github.io/bio/) CS PhD student.

**Comment by emeryberger** [comment]
Thanks!

**Pull request merged**